specification/schemas/Return.json

```diff
@@ -44,6 +44,11 @@
         "relationships": {
           "type": "object",
           "additionalProperties": false,
+          "required": [
+            "return_method",
+            "shop",
+            "status"
+          ],
```

Comment:
Let's define these requirements on the `ReturnResponse` and also on the POST and PATCH endpoint definitions.

The reason for this is more flexibility. If we make `status` required here, then the POST/PATCH body is going to require this relationship as well, while it's actually being set by our API after saving the resource.